src/ui/hooks/useMonaco.ts

```diff
@@ -60,14 +60,6 @@ const useMonaco = (Monaco: typeof import('../../editor/modules/monaco'), editorR
     setModels((models) => {
       return [...models, model];
     });
-    const openFrontmatter = models[0].findNextMatch('---\n', new Position(0, 0), false, false, ' \t\n', true);
-    const end = openFrontmatter.range.getEndPosition();
-    models[0].applyEdits([
-      {
-        range: new Range(end.lineNumber, end.column, end.lineNumber, end.column),
-        text: `import ${basename} from '@/${basename}.astro';\n`,
-      },
-    ]);
```

Comment:
I know this is buggy, but I don't think users would know to do `import Component from "@/Component.astro"` or `import Component from "../components/Component.astro"` without a hint from us. Any ideas on how we could do that?

Reply:
how about adding comment syntax kinda guide for developer? @natemoo-re 

```models[0].applyEdits([
      {
        range: new Range(end.lineNumber, end.column, end.lineNumber, end.column),
        text: `// import ${basename} from '@/${basename}.astro';\n`,
      },
    ]);```

Reply:
<img width="554" alt="Screen Shot 2021-09-17 at 06 05 39" src="https://user-images.githubusercontent.com/11748183/133696643-b6883a0c-d0ae-4d34-84eb-6051d6958aa7.png">

this is such a hint. i just revert back auto import with commented syntax

Reply:
I’m slightly more-inclined to just adding the import statements. If the lines will be there one way or the other, why not just import them (this is just a REPL after all)? They can be unused.

Reply:
@drwpow if we let the import syntax, in thr future when delete other component it might be facing an errror or astro compiler could take times to compile. This is such a mental mondel doing import why we do auto import only in repl but in text editor we can't do that

I just answered what @natemoo query how to give a hint by real adding new component

If astro new compiler is powerful to handle unused import, then it's awesome. No concern with auto import syntax

Reply:
Was wondering about this when plussing a component in the repl editor just now, with subsequent immediate delete.

How about a yes/no popup that asks if you want this component included at the top of Page.astro when plussed, and subsequently ask if you want the reference removed when trashed? That would make the behavior more transparent to the user instead of what happens now which is `✘ [ERROR] [plugin virtualfs] File \"/src/components/Component.astro\" does not exist ...`.

Could be confused for the editor being poorly coded and breaking unrelated to user action because of unseen magic.